sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -8887,6 +8963,366 @@ prefetch(const T *p, PropertyListT props = {}) {
   prefetch<T, VS>(p, 0, Mask, props);
 }
 
+/// template <typename T, int N, int VS, typename AccessorT,
+///           typename PropertyListT = empty_properties_t>
+/// void prefetch(AccessorT acc, simd<uint32_t, N / VS> byte_offsets,
```

Comment:
Major issue: it is not necessarily uint32_t. In stateless mode it is Ok to pass uint64_t-element vector byte_offsets